packages/mccs/mccs.1.1+10/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: [
+  "Claude Michel <claude.michel@unice.fr>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "http://www.i3s.unice.fr/~cpjm/misc/"
+bug-reports: "https://github.com/AltGr/ocaml-mccs/issues"
+license: "LGPL-2.1 with OCaml linking exception, BSD-3-clause, GPL-3.0"
+dev-repo: "git+https://github.com/AltGr/ocaml-mccs.git"
+build: [
+  ["dune" "build" "-p" name]
+  ["sh" "-c" "dune build @settests --auto-promote || true"] {with-test}
```

Comment:
Shouldn't we need a `--profile` here?

Reply:
Forwarding to @dra27 :grin:

Reply:
It shouldn't make a difference (at some point I should switch the tests to use the newer promotion system, but that's a battle for another day) as all this does is run an OCaml script, but it would do harm to have `--profile=release`, I agree